frontend/packages/ux-editor/src/containers/EditContainer.tsx

```diff
@@ -207,6 +208,7 @@ export function EditContainer(props: IEditContainerProps) {
           )}
           {isPreviewable && (
             <Button
+              aria-label={t("general.preview")}
```

Comment:
Very nitpicky here, but there is a mix of `'` and `"` for strings here, we generally use `'` in this project.